api/go.mod

```diff
@@ -1,18 +1,23 @@
 module github.com/cosmos/cosmos-sdk/api
 
-go 1.17
+go 1.18
 
 require (
 	github.com/cosmos/cosmos-proto v1.0.0-alpha7
 	github.com/gogo/protobuf v1.3.2
-	google.golang.org/genproto v0.0.0-20211223182754-3ac035c7e7cb
+	google.golang.org/genproto v0.0.0-20220222213610-43724f9ea8cf
 	google.golang.org/grpc v1.45.0
 	google.golang.org/protobuf v1.28.0
 )
 
 require (
 	github.com/golang/protobuf v1.5.2 // indirect
-	golang.org/x/net v0.0.0-20210405180319-a5a99cb37ef4 // indirect
-	golang.org/x/sys v0.0.0-20210510120138-977fb7262007 // indirect
-	golang.org/x/text v0.3.5 // indirect
+	github.com/google/go-cmp v0.5.7 // indirect
+	golang.org/x/net v0.0.0-20211208012354-db4efeb81f4b // indirect
+	golang.org/x/sys v0.0.0-20220114195835-da31bd327af9 // indirect
+	golang.org/x/text v0.3.7 // indirect
 )
+
+replace github.com/gogo/protobuf => github.com/regen-network/protobuf v1.3.3-alpha.regen.1
+
+replace github.com/cosmos/cosmos-sdk => ../
```

Comment:
don't think this is needed here?

Reply:
I can try removing it-- I put it in a number of other places, too.